projects/scion/components.internal/package.json

```diff
@@ -26,10 +26,10 @@
     "tslib": "^2.4.0"
   },
   "peerDependencies": {
-    "@angular/common": "^15.0.0",
-    "@angular/core": "^15.0.0",
-    "@angular/cdk": "^15.0.0",
-    "@scion/components": "^15.0.0",
+    "@angular/common": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@angular/cdk": "^16.0.0",
+    "@scion/components": "^16.0.0",
     "@scion/toolkit": "^1.0.0",
     "rxjs": "^7.5.0"
```

Comment:
I would change it to `^7.8.0` because this is the minimal version Angular requires.